scassandra/src/main/scala/com/evolutiongaming/scassandra/Metadata.scala

```diff
@@ -138,6 +138,8 @@ object KeyspaceMetadata {
       val replication = {
         Sync[F].delay { ToScala.from(keyspaceMetadata.getReplication).toMap }
       }
+
+      val userTypes = ToScala.from(keyspaceMetadata.getUserTypes).toList
```

Comment:
Exposing `UserType` in `scassandra` might not be what library author intended. It has quite unsafe API. I would ask @t3hnar if it is okay.

Reply:
I have the next purpose:
I want to use UserType to create UDT(User Defined Type) and correctly store information instead of rejecting type and convert entity to Json.

Reply:
Entity Example:

```
{
field_name : [{field_name_1 : field_name_value1, field_name_2 : field_name_value2, ...}]
}
```

Reply:
In current moment we didn't cover such cases, unfortunately.
Because 'Scassandra' it's a wrapper around java-cassandra-driver and we should use the next code.

`
val state: [B <: SettableData[B]] = ???
state
.setString("field", "value")
.setList("field", ???)
`

Reply:
I can cover my goal via create UserType on my side, but it's a duplication because all necessary information already stored on Cassandra metadata level.

Reply:
> Exposing UserType in scassandra might not be what library author intended

I support this comment by @rtar, however I'd like to stay on the practical size: hence I'm find with adding this method…

Reply:
@t3hnar I'm glad to hear that.